composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "atelierdisko/coupon_code",
+    "name": "HYP3/coupon_code",
```

Comment:
There should be no changes to the `composer.json` file necessary. I guess this is a leftover from local development.